static/app/views/dashboardsV2/detail.tsx

```diff
@@ -378,17 +354,15 @@ class DashboardDetail extends Component<Props, State> {
       case DashboardState.PREVIEW:
       case DashboardState.CREATE: {
         if (modifiedDashboard) {
-          // Allow duplicate dashboard names when in preview mode
-          createDashboard(api, organization.slug, modifiedDashboard, this.isPreview).then(
+          let newModifiedDashboard = modifiedDashboard;
+          if (organization.features.includes('dashboard-grid-layout')) {
+            newModifiedDashboard = constructDashboardWidgetsWithLayout(
+              modifiedDashboard,
+              layout
+            );
+          }
+          createDashboard(api, organization.slug, newModifiedDashboard).then(
```

Comment:
this.isPreview is missing from the createDashboard call

Reply:
Good catch, thanks!